src/Form/EntityVersionSettingsForm.php

```diff
@@ -257,7 +257,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       }
     }
 
-    $this->messenger->addStatus($this->t('The Version history configuration has been saved.'));
+    $this->messenger->addStatus($this->t('The Entity Version configuration has been saved.'));
```

Comment:
```suggestion
    $this->messenger->addStatus($this->t('The Entity version configuration has been saved.'));
```
Let's not use camel case for names.